tensorflow/java/BUILD

```diff
@@ -40,6 +41,31 @@ filegroup(
     ],
 )
 
+java_plugin(
+    name = "operator_plugin",
+    generates_api = True,
+    processor_class = "org.tensorflow.processor.operator.OperatorProcessor",
+    visibility = ["//visibility:public"],
+    deps = [":operator_plugin_library"],
+)
+
+java_library(
+    name = "operator_plugin_library",
+    srcs = [":operator_plugin_sources"],
```

Comment:
Why bother with the filegroup here instead of say `srcs = glob(["src/gen/java/org/tensorflow/processor/**/*.java"])`?